R/pkg/R/SQLContext.R

```diff
@@ -90,19 +90,18 @@ createDataFrame <- function(sqlContext, data, schema = NULL, samplingRatio = 1.0
       if (is.null(schema)) {
         schema <- names(data)
       }
-      n <- nrow(data)
-      m <- ncol(data)
-      # get rid of factor type
-      dropFactor <- function(x) {
+      # get rid of factor type and adjust for lists
```

Comment:
Recommend the comment:
# Because parallelize() parallelizes a data.frame by columns, so transpose the data.frame here before passing it to parallelize() so that each row in the data.frame corresponds to a row in the result DataFrame.